### PR TITLE
docs(readme): comply new policy template INTER-282

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,11 @@
```
Comment (JuroUhlar) [inline README.md:36]: Can we align the introductory paragraph with other SDKs, using the [Readme guidelines](https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-9556ed587c0749b29f32a18017413cbc?pvs=4#9c47b1318fa647d6af91254004bdb9b8)?

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Orkuncakilkaya) [reply README.md:36]: Sure, I'll work on it 👍

**Review by JuroUhlar — APPROVED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**